tools/rust_api/include/kuzu_rs.h

```diff
@@ -186,6 +205,21 @@ template<typename T>
 std::unique_ptr<kuzu::common::Value> create_value(const T value) {
     return std::make_unique<kuzu::common::Value>(value);
 }
+inline std::unique_ptr<kuzu::common::Value> create_value_decimal(int64_t high, uint64_t low,
```

Comment:
I initially wrote decimal thinking of it as a front end for integers, which unfortunately makes it difficult to interact with in some parts of teh code. We could create a decimal type `decimal_t` containing `int128_t val, uint32_t precision, uint32_t scale` to construct decimal values for C++ API users. Changing value creation to that of LogicalType creation works, but it's a lot more work than adding `decimal_t`.

Reply:
That sounds reasonable to me.